client/src/components/personsList/personsListController.js

```diff
@@ -0,0 +1,10 @@
+function UsersListController($http) {
```

Comment:
Suggest to rewrite it to Class notation. It will be easy while it's a simple controller